.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 ci:
     autoupdate_schedule: 'quarterly'
-default_language_version:
-    python: python3.9
+# default_language_version:
+#     python: python3.9
```

Comment:
For pre-commit to work locally for you this needs to be commented out, but for it to work in CI this needs to exist. 
Two options:
1. Uninstall pre-commit locally (https://stackoverflow.com/questions/44334746/how-to-uninstall-pre-commit) as doing the checks in CI is probably sufficient and potentially easier.
2. Comment this line locally but do not commit